### PR TITLE
Comments on git pull requests lesson

### PR DESCRIPTION
**Pull request by chrisburr**
I was flying as I reviewed this so I've written this as a PR instead, revert/modify the changes as you see fit.

What git lesson would be complete without some very complicated branching to write it! 😆

### CODE REVIEW

File: git/09-pullrequests.md [hunk 1]
```diff
@@ -150,11 +147,8 @@ repository. The second one is named after your CERN account and you have created
 it explicitly with `git remote add`.
 {% endchallenge %}
 
-Note that learning how to work with several remotes is essential when using Pull
```
Comment (chrisburr) [inline git/09-pullrequests.md]: This isn't correct as the same can be done using personal or feature branches instead. Having used both methods, I think they both have different merits.

Comment (dberzano) [reply git/09-pullrequests.md]: I see, although normally, i.e. "outside HEP", people work on forks, not feature branches on the main repo :-) Let me rephrase it.

**Comment by dberzano** [comment]
OK I hadn't realized this is a PR to my PR to your PR, I am lost in recursion :-) Let me have a look.

File: git/09-pullrequests.md [hunk 2]
```diff
@@ -351,12 +342,11 @@ Now, sit back and relax!
...
-Once a Merge Request is opened, it behaves like a social network. Repository
-owners are notified (usually via email), and they (or even somebody else,
-according to the repository's permissions) can add comments and requests before
-accepting your changes.
+Once a Merge Request is opened the repository owners are notified (usually via
+email), and they (or even somebody else, depending on the repository's
+permissions) can add comments and requests before accepting your changes.
 
-A Merge Request is given a progressive number. Here's the merge request link:
+Merge Requests is given a sequential number. Here is an example of a merge request:
```
Comment (dberzano) [inline git/09-pullrequests.md]: Isn't it "Merge Requests ARE given a sequential number"?

File: git/09-pullrequests.md [hunk 3]
```diff
@@ -402,9 +392,9 @@ To ssh://gitlab.cern.ch:7999/dberzano/starter-kit-example.git
    256c9b6..d09c134  master -> master
 ```
 
-Voilà, it's done: you have just added more commits to your Merge Request! Let's
-see on the Web all the commits you have added. Go to the Merge Request webpage
-and click on the **Commits** tab at the bottom, here's a direct link:
+Voilà, it's done: you have just added more commits to your Merge Request! Lets
```
Comment (dberzano) [inline git/09-pullrequests.md]: I think "Let's" was correct, was it?

**Comment by dberzano** [comment]
OK I am done, I agree with all your changes (how couldn't I? You're a native speaker haha!) and I also feel ashamed for some basic mistakes (er, "learned")... can you have a look at the couple of comments I have added?

File: git/09-pullrequests.md (hunk 2 above)
Comment (chrisburr) [reply git/09-pullrequests.md]: Yes, well spotted!

File: git/09-pullrequests.md (hunk 3 above)
Comment (chrisburr) [reply git/09-pullrequests.md]: And this is why you shouldn't blindly trust native speakers...I've learnt something today!

File: git/09-pullrequests.md (hunk 2 above)
Comment (chrisburr) [reply git/09-pullrequests.md]: I'm not sure what I was doing there, I've rewritten it slightly.

File: git/09-pullrequests.md (hunk 1 above)
Comment (chrisburr) [reply git/09-pullrequests.md]: This is definitely the case for open source projects, I'm not so sure that it dominates in private repositories. But it doesn't matter here, I'll let you merge this and rephrase it as you see fit.
